java/impl/aws/src/main/java/co/worklytics/psoxy/S3Handler.java

```diff
@@ -60,6 +60,13 @@ StorageEventResponse process(String importBucket, String sourceKey, StorageHandl
         StorageEventResponse storageEventResponse;
         S3Object s3Object = s3Client.getObject(new GetObjectRequest(importBucket, sourceKey));
 
+        if (storageHandler.hasBeenSanitized(s3Object.getObjectMetadata().getUserMetadata())) {
```

Comment:
Can s3Object.getObjectMetadata() returns null ?

Reply:
reasonable to be defensive and check for it; nothing in AWS's doc of the interface specifies that it can't be